client/my-sites/plans/ecommerce-trial/wooexpress-plans/style.scss

```diff
@@ -28,7 +28,53 @@
 		font-family: $sans;
 	}
 
-	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price {
+	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price,
+	.plan-features-2023-grid__plan-comparison-grid-container .plan-comparison-grid .plan-price {
 		font-family: $sans;
 	}
+
+	.plan-price .plan-price__currency-symbol {
+		font-size: 1.5rem;
+	}
+
+	.plan-features-2023-grid__plan-comparison-grid-container .plan-comparison-grid .wp-brand-font {
+		display: none;
+	}
+
+	.plan-features-2023-grid__table.has-2-cols {
+		margin-left: auto;
+		margin-right: auto;
+		max-width: 800px;
+	}
+}
+
+.enterprise-ecommerce__banner {
+	align-items: center;
+	border: 1px solid var(--color-border-subtle);
+	border-radius: 4px;
+	display: flex;
+	gap: 1em;
+	justify-content: space-between;
+	margin: 0.5em auto 2em;
+	max-width: calc(800px - 3em);
+	padding: 1em 1.5em 1.5em;
+
+	.enterprise-ecommerce__title {
+		font-size: 1.5rem;
+		margin-bottom: 0.5em;
+		margin-top: 0;
+	}
+
+	.enterprise-ecommerce__subtitle {
```

Comment:
The subtitle and button spacing is a bit close. Let's add some padding/margin.

![Screenshot 2023-06-15 at 12 34 10](https://github.com/Automattic/wp-calypso/assets/4344253/37a0e9ee-bffb-4b02-bfc3-3084a05ff09f)

Reply:
Fixed!